tests/test_proxy.py

```diff
@@ -15,7 +15,6 @@ class TestImplemIterator(PythonJavaClass):
         'java/util/ListIterator', ]
 
     def __init__(self, collection, index=0):
-        super(TestImplemIterator, self).__init__()
```

Comment:
Can you just explain me why this is not necessary? (i always call super)
Will it break current code available today that calls super() in init ?

Reply:
I removed that line to test what happens if the user forgets to call `super`. Having a super call there does not break anything, but removing it means f0cd0f4 isn't covered by tests.